lib/master.js

```diff
@@ -267,6 +287,8 @@ class Master extends BaseService {
                     case 'ratelimiter_counters':
                         return this._ratelimiter &&
                                 this._ratelimiter.updateCounters(msg.value);
+                    case 'prom-client:getMetricsRes':
+                        return;
```

Comment:
I guess it's needed as it's an internal communication mode of the prometheus lib, but it's confusing. Let's add a comment.